### ISSUE TITLE
Link tags in React Router acting as normal anchor tags in loop mode

### ISSUE THREAD

**Issue by JoshMoxey** [kidjp85/react-id-swiper#131]
In loop mode, the "duplicate" slides that have Link tags (from React Router)behave as normal anchor tags and reload the full page when clicked on. Normal swiper slides (non duplicates) act as the Link tags should.

I'm on Swiper version: 1.5.6. Thanks :)

**Comment by JoshMoxey** [comment]
I can confirm that upon further testing, it's not just Link tags. It's also libraries like react-truncate, for example, that have their tags ignored.

**Comment by JoshMoxey** [comment]
Any update on this?

**Comment by mantagen** [comment]
I experienced a potentially similar problem where I had JSX with event listeners which didn't work in the duplicate slides.

**Comment by SaeedSheikhi** [comment]
same issue when ` loop: true`

**Comment by RumenTech** [comment]
Thanks for the great plugin.

I can confirm this is reproducible.
When you check in React devtools extension, only original set of slides are present.
It seems like duplicate slides are just being created by `swiper` library.

**Comment by kidjp85** [comment]
Hi @all , sorry for my late response. So busy recently. When `loop: true`,  Swiper will copy 2 DOM nodes at the beginning and end, also events are not be copied, so any event bound is invalid.

**Comment by kidjp85** [comment]
I made a quick fix here [here](https://codesandbox.io/s/4355qo54z4) . To be honest, I don't want to do in this way, but Swiper has limitation in loop mode.  

My solutions for this issue:
1. Make swiper ref 
2. Replace `Link` with HOC `withRouter` to use `history.push`
3. In `componentDidUpdate`, add click event for duplicated slides

PS: I wrote this with `recompose`, if u guys have any question , feel free to ask.
